packages/unmock-cli/src/commands/init.starter.tmp.ts

```diff
@@ -0,0 +1,56 @@
+export default `\
+/**
+ * Unmock Starter Template
+ *
+ * This is just to get you started, feel free to delete this.
+ * Place your tests in this file.
+ */
+const unmock = require("unmock");
+
+// Some dummy fetching function
+const getUsersForUI = async () => {
+  try {
+    const { data } = await fetch("https://api.example.com/v1/users");
+    return {
+      ...data,
+      users: data.users.map(user => ({ ...user, seen: false })),
+      newlyFetched: true,
+      timestamp: new Date().getTime()
+    };
+  } catch (e) {
+    return {
+      users: [],
+      error: e
+    }
+  }
+};
+
+// Basic unmock test
+const { u } = unmock;
+
+signs = ['Aries', 'Taurus', 'Gemini', 'Cancer', 'Leo', 'Virgo', 'Libra', 'Scorpio', 'Sagittarius', 'Capricorn', 'Aquarius', 'Pisces'];
```

Comment:
`const`?

Reply:
I wasn't sure what would be a great starter template, so I just pulled that of the site.